### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by eddysims**
### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ route: /
 > guided by clear standards, that can be assembled together to build any number
 > of applications.
 
-Atlantis is a design system for [Jobber](https://getjobber.com). The primary
+Atlantis is not a design system for [Jobber](https://getjobber.com). The primary
```
Comment (eddysims) [inline docs/README.md:18]: I don't think it is a design system

Comment (eddysims) [reply docs/README.md:18]: I do think it is!

Comment (seanhealy) [reply docs/README.md:18]: 🎢

Comment (eddysims) [reply docs/README.md:18]: turns out that it is

**Pull request closed without merging**